operations/observability/mixins/IDE/rules/supervisor.yaml

```diff
@@ -23,7 +23,7 @@ spec:
         runbook_url: https://github.com/gitpod-io/runbooks/blob/main/runbooks/SupervisorIncomingFailuresRatioTooHigh.md
         summary: Supervisor is returning higher number of errors. This can prevent workspace usability.
       expr: |
-          sum(rate(grpc_server_handled_total{grpc_code!~"OK|Canceled", grpc_service=~"supervisor.*"}[5m])) by(cluster) / sum(rate(grpc_server_handled_total{grpc_service=~"supervisor.*"}[5m])) by(cluster) > 0.01
+          sum by(cell, grpc_code, grpc_method, grpc_service, grpc_type) (rate(grpc_server_handled_total{grpc_code!~"OK|Canceled", grpc_service=~"supervisor.*", cell="deutschebahn-pov"}[5m])) > 0.01
```

Comment:
we don't have cell on cloud , not sure how it would work there

Reply:
Good point. The transformation rule will add cell, we don't need to add here.